### PR TITLE
Launch josm with osmxml

### PR DESCRIPTION
**Pull request by feenster**

### CODE REVIEW

File: client/app/project/project.controller.js [hunk 1]
```diff
@@ -1073,6 +1073,17 @@
                         };
                         editorService.sendJOSMCmd('http://127.0.0.1:8111/imagery', imageryParams);
                     }
+
+                    //load task squares into JOSM
+
+
+                    var importParams = {
+                        //url: encodeURIComponent('http://localhost:3000/assets/img/osmxml3.osm'),
+                        //url: encodeURIComponent('http://tasking-manager-staging.eu-west-1.elasticbeanstalk.com/api/v1/project/2666/tasks-as-osm-xml?tasks=34,99,98'),
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: remove dev comments

File: client/app/services/editor.service.js [hunk 2]
```diff
@@ -159,5 +160,26 @@
...
+            var osmUrl = configService.tmAPI + '/project/' + projectId + '/tasks-as-osm-xml?tasks=' + taskIds;
+            // If it is not a full path, then it must be relative and for the GPX callback to work it needs
+            // a full URL so get the current host and append it
+            // Check if it is a full URL
+            var fullUrl = osmUrl.indexOf('http');
+            if (fullUrl == -1){
+                // Not a full URL - so add the absolute part
+                osmUrl = $location.protocol() + '://' + $location.host() + osmUrl;
+            }
+            console.log(osmUrl);
```
Comment (LindaAlblas) [inline client/app/services/editor.service.js]: remove console.log

File: client/app/project/project.controller.js (hunk 1 above)
Comment (feenster) [inline client/app/project/project.controller.js]: done

File: client/app/services/editor.service.js (hunk 2 above)
Comment (feenster) [inline client/app/services/editor.service.js]: done

**Pull request merged**